### ISSUE TITLE
Text area unclickable when tabulator height set to a percentage

### ISSUE THREAD

**Issue by thearchitect3344** [tabulator-tables/tabulator#4142]
**Describe the bug**
 When adding a table with a height set to a percentage, the text area editor becomes unclickable. It appears that tabulator is throwing the error "ResizeObserver loop limit exceeded" 

**Tabulator Info**
5.4.4

**Working Example**
https://jsfiddle.net/kmtg6pf1/

**To Reproduce**
1. Go to https://jsfiddle.net/kmtg6pf1/
2. Click on the first row of the column "info"
3. Notice the column cannot be edited

**Expected behavior**
The column can be edited when clicked

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser chrome
 - Version 110

**Comment by Jojoshua** [comment]
Hey @olifolkerd, this seems to be an issue for quite a few of us. If we wanted to contribute a PR for this, could you give some pointers of where to begin?

**Comment by AVerdolini** [comment]
I confirm the bug. the problem can be temporarily resolved by setting a fixed table height. 
This problem appears when you have the table height in percentage or "auto".

Tabulator Info
6.2.0
Here's another example:
 https://jsfiddle.net/verdonfly/9753zbjp/
to simulate it just change the value of "height" and try to modify the cell.

**Comment by Jojoshua** [comment]
@azmy60 I saw 6.4 dropped but tested out the original fiddle in this issue with 6.4 and still seeing the same behavior.

**Comment by rathboma** [comment]
@Jojoshua I'm planning for a June bugfix release and I'm going to try and include a fix for this issue (if we can replicate it)

**Comment by Jojoshua** [comment]
> [@Jojoshua](https://github.com/Jojoshua) I'm planning for a June bugfix release and I'm going to try and include a fix for this issue (if we can replicate it)

@rathboma That would be great, been waiting for this for 3 years, let me know if you can't reproduce

**Comment by Jojoshua** [comment]
@rathboma @not-night-but I can't reopen this but wanted to follow-up. It seems as if the original reproduction (with the updated dist files) still happening. See https://jsfiddle.net/qj8ouvx2/3/ 

Clicking in the cell still produces a resize observer error

<img width="560" height="575" alt="Image" src="https://github.com/user-attachments/assets/7eee123c-2327-4e42-98c4-e7f978d72303" />

**Comment by not-night-but** [comment]
@Jojoshua Hey! It looks like the PR was merged into master, but it hasn't been included in a release yet. Are you only experiencing this in the latest release, or have you tried master? We are planning to do another release in the next couple weeks

**Comment by Jojoshua** [comment]
@not-night-but oh! No I was using 6.5.2 and thought it was included. We're not able to pull until it hits npm, we will try it again when released. Thanks so much for tracking this down.

**Comment by not-night-but** [comment]
@Jojoshua Yeah so sorry for the confusion. Github auto closes the linked issues when a PR is merged to master

**Comment by rathboma** [comment]
I'm going to close the issue as the problem is fixed -- but you are correct that this is not in the latest release, it will be available in v 6.5.3
